contracts/bonding/BondingManager.sol

```diff
@@ -248,50 +253,44 @@ contract BondingManager is IBondingManager, Manager {
 
     /*
      * @dev Unbond delegator's current stake. Delegator enters unbonding state
+     * @param _amount Amount of tokens to unbond
      */
-    function unbond() external whenSystemNotPaused returns (bool) {
-        // Current round must be initialized
-        require(roundsManager().currentRoundInitialized());
-
-        Delegator storage del = delegators[msg.sender];
-
+    function unbond(uint256 _amount)
```

Comment:
Is _amount used in this method? It looks like it unbonds the del.bondedAmount.

Reply:
Ah I started thinking about unbonding specific amounts of stake, but didn't end up implementing it for this spike. I'll remove it for now